frontend/src/components/CategorySelect.tsx

```diff
@@ -112,11 +121,24 @@ const CategorySelect: FC<ICategorySelectProps> = ({
           '& .MuiSelect-icon': { color: '#f2f2f2' },
         }}
       >
-        {categories.map((category, i) => (
-          <MenuItem key={i} value={category.id} sx={{ fontSize: '0.85em', fontWeight: 600 }}>
-            {category.name}
-          </MenuItem>
-        ))}
+        {categories.map((category, i) => {
```

Comment:
We should use the old mapping function (the one which is present in dev, and add another menu item below it for new category

Reply:
done